lib/service/invoices.go

```diff
@@ -168,6 +161,19 @@ func createLnRpcSendRequest(invoice *models.Invoice) (*lnrpc.SendRequest, error)
 	}, nil
 }
 
+func calcFeeLimit(invoice *models.Invoice) lnrpc.FeeLimit {
+	limit := int64(10)
```

Comment:
put these values here based on issue description
is there a way to test this somehow? currently test nodes are not responding, so I can not run existing ones, when that is stable i will verify and potentially add more

Reply:
Are the test nodes working again? @kiwiidb

Reply:
Tests have passed now.

Reply:
I think a test for the calcFeeLimit function would be good.

Reply:
@bumi i added couple unit tests for this function, please check

Reply:
OK